tensorflow/lite/micro/kernels/add_test.cc

```diff
@@ -106,6 +132,26 @@ void TestAddFloat(int* input1_dims_data, const float* input1_data,
                      ElementCount(*output_dims), activation);
 }
 
+void TestAddInt32(int* input1_dims_data, const int32_t* input1_data,
+                  int* input2_dims_data, const int32_t* input2_data,
+                  int* output_dims_data, const int32_t* expected_output,
+                  TfLiteFusedActivation activation, int32_t* output_data) {
+  TfLiteIntArray* input1_dims = IntArrayFromInts(input1_dims_data);
+  TfLiteIntArray* input2_dims = IntArrayFromInts(input2_dims_data);
+  TfLiteIntArray* output_dims = IntArrayFromInts(output_dims_data);
+
+  constexpr int inputs_size = 2;
+  constexpr int outputs_size = 1;
+  constexpr int tensors_size = inputs_size + outputs_size;
+  TfLiteTensor tensors[tensors_size] = {
+      CreateTensor(input1_data, input1_dims),
+      CreateTensor(input2_data, input2_dims),
+      CreateTensor(output_data, output_dims),
+  };
+  ValidateAddGoldensInt(tensors, tensors_size, expected_output, output_data,
```

Comment:
Is there any issue using `ValidateAddGoldens()` here instead? `TF_LITE_MICRO_EXPECT_NEAR` skips the tolerance if the two values are equal. If not, I think it's best to avoid adding yet another helper function.

Reply:
Good call, I assumed it did a floating point comparison but looking at the macro it should be safe with int32.